npm build failure due to no-unexpexcted-multiline eslint rule

**Problem**: `npm run build` fails due to the eslint error `no-unexpected-multiline` rule activated via the recommended eslint property (https://eslint.org/docs/rules/no-unexpected-multiline).

**My config**:

* node -v: v10.20.0
* npm -v: 6.14.4


**Use case with logs:**

```sh
BlinkWebUI git/master
❯ npm run build

> blink-web-ui@1.0.0 build /Users/mcmoe/dev/BlinkWebUI
> NODE_ENV=production webpack

Hash: 4c0c7acc9556411beeda
Version: webpack 4.44.1
Time: 35999ms
Built at: 08/27/2020 4:34:27 PM
 4 assets
Entrypoint main = main.css main.bundle.js main.css.map main.bundle.js.map
  [7] ./node_modules/@material-ui/utils/esm/index.js + 8 modules 8 KiB {0} [built]
      |    9 modules
 [34] ./src/application/layouts/main/MainLayoutStyle.module.scss 726 bytes {0} [built]
 [58] ./src/application/pages/recordings/RecordingsPage.module.scss 469 bytes {0} [built]
 [63] ./node_modules/@material-ui/styles/esm/index.js + 5 modules 11 KiB {0} [built]
      |    6 modules
 [83] ./node_modules/@material-ui/core/esm/styles/defaultTheme.js + 16 modules 26.8 KiB {0} [built]
      |    17 modules
 [84] ./src/application/pages/login/LoginPageStyle.module.scss 292 bytes {0} [built]
 [98] ./src/application/pages/cameras/CamerasPageStyle.module.scss 233 bytes {0} [built]
 [99] ./src/application/pages/networks/NetworksPageStyle.module.scss 229 bytes {0} [built]
[119] (webpack)/buildin/global.js 475 bytes {0} [built]
[154] ./src/application/pages/sync-modules/SyncModulesPageStyle.module.scss 161 bytes {0} [built]
[158] ./src/application/i18n/resources.json 1.4 KiB {0} [built]
[159] ./node_modules/@material-ui/core/esm/Menu/Menu.js + 2 modules 38 KiB {0} [built]
      |    3 modules
[171] multi ./src/index.tsx 28 bytes {0} [built]
[238] ./src/index.scss 39 bytes {0} [built]
[239] ./src/index.tsx + 98 modules 491 KiB {0} [built] [8 warnings] [4 errors]
      | ./src/index.tsx 455 bytes [built]
      | ./src/App.tsx 4.45 KiB [built]
      | ./src/application/config/ServiceLocator.ts 2.36 KiB [built]
      | ./src/application/theme/theme.tsx 519 bytes [built]
      | ./src/application/layouts/main/MainLayout.tsx 12 KiB [built] [1 warning]
      | ./src/application/i18n/i18n.ts 480 bytes [built]
      | ./src/application/utils/SimplePubSub.ts 2.79 KiB [built] [1 warning]
      | ./src/infrastructure/repositories/impl/blink/AuthRepositoryImpl.ts 1.59 KiB [built]
      | ./src/infrastructure/repositories/impl/blink/CameraRepositoryImpl.ts 2.19 KiB [built]
      | ./src/infrastructure/repositories/impl/blink/MediaRepositoryImpl.ts 2.26 KiB [built]
      | ./src/infrastructure/repositories/impl/blink/NetworkRepositoryImpl.ts 2.97 KiB [built]
      | ./src/infrastructure/repositories/impl/blink/SyncModuleRepositoryImpl.ts 1.79 KiB [built]
      | ./src/application/config/config.ts 291 bytes [built]
      | ./src/application/pages/login/LoginPage.tsx 6.46 KiB [built]
      | ./src/application/pages/recordings/RecordingsPage.tsx 7.08 KiB [built] [1 warning]
      |     + 84 hidden modules
    + 311 hidden modules

WARNING in ./src/application/components/async-switch/AsyncSwitchComponent.tsx
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/components/async-switch/AsyncSwitchComponent.tsx
  42:32  warning  'prevState' is defined but never used  @typescript-eslint/no-unused-vars

✖ 1 problem (0 errors, 1 warning)

 @ ./src/application/pages/networks/NetworksPage.tsx 44:0-86 144:105-125
 @ ./src/application/layouts/main/MainLayout.tsx
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi ./src/index.tsx

WARNING in ./src/application/components/recoding/RecordingComponent.tsx
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/components/recoding/RecordingComponent.tsx
   85:40  warning  Missing return type on function  @typescript-eslint/explicit-function-return-type
  116:30  warning  Missing return type on function  @typescript-eslint/explicit-function-return-type
  133:30  warning  Missing return type on function  @typescript-eslint/explicit-function-return-type

✖ 3 problems (0 errors, 3 warnings)

 @ ./src/application/pages/recordings/RecordingsPage.tsx 39:0-78 135:46-64
 @ ./src/application/layouts/main/MainLayout.tsx
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi ./src/index.tsx

WARNING in ./src/application/components/video-player/VideoPlayerComponent.tsx
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/components/video-player/VideoPlayerComponent.tsx
  19:26  warning  Missing return type on function  @typescript-eslint/explicit-function-return-type

✖ 1 problem (0 errors, 1 warning)

 @ ./src/application/components/recoding/RecordingComponent.tsx 36:0-72 134:64-84
 @ ./src/application/pages/recordings/RecordingsPage.tsx
 @ ./src/application/layouts/main/MainLayout.tsx
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi ./src/index.tsx

WARNING in ./src/application/layouts/dashboard-page/DashboardPageLayout.tsx
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/layouts/dashboard-page/DashboardPageLayout.tsx
  23:5  warning  Missing return type on function  @typescript-eslint/explicit-function-return-type

✖ 1 problem (0 errors, 1 warning)

 @ ./src/application/pages/networks/NetworksPage.tsx 40:0-83 129:46-65
 @ ./src/application/layouts/main/MainLayout.tsx
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi ./src/index.tsx

WARNING in ./src/application/layouts/main/MainLayout.tsx
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/layouts/main/MainLayout.tsx
   56:42  warning  Forbidden non-null assertion     @typescript-eslint/no-non-null-assertion
   92:5   warning  Missing return type on function  @typescript-eslint/explicit-function-return-type
  168:37  warning  Missing return type on function  @typescript-eslint/explicit-function-return-type

✖ 3 problems (0 errors, 3 warnings)

 @ ./src/App.tsx 40:0-63 62:89-99
 @ ./src/index.tsx
 @ multi ./src/index.tsx

WARNING in ./src/application/pages/cameras/CamerasPage.tsx
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/pages/cameras/CamerasPage.tsx
  113:30  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

✖ 1 problem (0 errors, 1 warning)

 @ ./src/application/layouts/main/MainLayout.tsx 56:0-58 148:49-60
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi ./src/index.tsx

WARNING in ./src/application/pages/recordings/RecordingsPage.tsx
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/pages/recordings/RecordingsPage.tsx
  52:42  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

✖ 1 problem (0 errors, 1 warning)

 @ ./src/application/layouts/main/MainLayout.tsx 55:0-67 142:49-63
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi ./src/index.tsx

WARNING in ./src/application/utils/SimplePubSub.ts
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/utils/SimplePubSub.ts
  21:54  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
  23:58  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
  27:9   warning  Forbidden non-null assertion              @typescript-eslint/no-non-null-assertion
  30:61  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
  32:13  warning  Forbidden non-null assertion              @typescript-eslint/no-non-null-assertion
  40:46  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
  43:17  warning  Forbidden non-null assertion              @typescript-eslint/no-non-null-assertion

✖ 7 problems (0 errors, 7 warnings)

 @ ./src/application/layouts/main/MainLayout.tsx 52:0-69 76:4-16 76:27-38 79:4-16 79:27-38 82:4-16 82:27-38 87:4-16 87:27-38
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi ./src/index.tsx

WARNING in configuration
The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/

ERROR in ./src/application/components/loading/LoadingComponent.tsx
Module Error (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/components/loading/LoadingComponent.tsx
  11:1  error  More than 1 blank line not allowed  no-multiple-empty-lines

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

 @ ./src/application/layouts/dashboard-page/DashboardPageLayout.tsx 34:0-73 63:44-60
 @ ./src/application/pages/networks/NetworksPage.tsx
 @ ./src/application/layouts/main/MainLayout.tsx
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi ./src/index.tsx

ERROR in ./src/application/components/ui-console/UiConsoleComponent.tsx
Module Error (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/components/ui-console/UiConsoleComponent.tsx
  34:1   error    More than 1 blank line not allowed  no-multiple-empty-lines
  70:39  warning  Forbidden non-null assertion        @typescript-eslint/no-non-null-assertion

✖ 2 problems (1 error, 1 warning)
  1 error and 0 warnings potentially fixable with the `--fix` option.

 @ ./src/application/layouts/main/MainLayout.tsx 53:0-104 85:6-24 85:37-55 254:102-120
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi ./src/index.tsx

ERROR in ./src/application/pages/networks/NetworksPage.tsx
Module Error (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/pages/networks/NetworksPage.tsx
   38:1   error    More than 1 blank line not allowed        no-multiple-empty-lines
  101:31  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

✖ 2 problems (1 error, 1 warning)
  1 error and 0 warnings potentially fixable with the `--fix` option.

 @ ./src/application/layouts/main/MainLayout.tsx 58:0-61 160:49-61
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi ./src/index.tsx

ERROR in ./src/application/pages/sync-modules/SyncModulesPage.tsx
Module Error (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/mcmoe/dev/BlinkWebUI/src/application/pages/sync-modules/SyncModulesPage.tsx
  37:1   error    More than 1 blank line not allowed        no-multiple-empty-lines
  53:30  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

✖ 2 problems (1 error, 1 warning)
  1 error and 0 warnings potentially fixable with the `--fix` option.

 @ ./src/application/layouts/main/MainLayout.tsx 57:0-71 154:49-64
 @ ./src/App.tsx
 @ ./src/index.tsx
 @ multi ./src/index.tsx
...
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! blink-web-ui@1.0.0 build: `NODE_ENV=production webpack`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the blink-web-ui@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mcmoe/.npm/_logs/2020-08-27T14_34_27_693Z-debug.log

BlinkWebUI git/master  38s
```

--- a/src/application/components/loading/LoadingComponent.tsx
+++ b/src/application/components/loading/LoadingComponent.tsx
@@ -8,7 +8,6 @@
 import React from 'react';
 import CircularProgress from '@material-ui/core/CircularProgress';
 
-
 export default class LoadingComponent extends React.PureComponent {
     render(): JSX.Element {
         return <CircularProgress />;

--- a/src/application/components/ui-console/UiConsoleComponent.tsx
+++ b/src/application/components/ui-console/UiConsoleComponent.tsx
@@ -31,7 +31,6 @@ export interface UiConsoleComponentState {
     alerts: UIConsoleAlert[];
 }
 
-
 export default class UiConsoleComponent extends React.PureComponent<{}, UiConsoleComponentState> {
     private static eventsToTypeMap: { [k in PubSubEvent]?: UIConsoleAlert['type'] } = {
         [PubSubEvent.UI_CONSOLE_ERROR]: UIConsoleAlertType.ERROR,

--- a/src/application/pages/networks/NetworksPage.tsx
+++ b/src/application/pages/networks/NetworksPage.tsx
@@ -35,7 +35,6 @@ interface NetworksPageProps extends WithTranslation {
     session: Session;
 }
 
-
 class NetworksPage extends React.PureComponent<PropsWithChildren<NetworksPageProps>, NetworksPageState> {
     networkRepository: NetworkRepository = Container.get(networkRepositoryToken);
 

--- a/src/application/pages/sync-modules/SyncModulesPage.tsx
+++ b/src/application/pages/sync-modules/SyncModulesPage.tsx
@@ -34,7 +34,6 @@ interface SyncModulesPageProps extends WithTranslation{
     session: Session;
 }
 
-
 class SyncModulesPage extends React.PureComponent<PropsWithChildren<SyncModulesPageProps>, SyncModulesPageState> {
     syncModuleRepository: SyncModuleRepository = Container.get(syncModuleRepositoryToken);
 

